.github/workflows/RELEASE_OOTB_IMAGE.yaml

```
@@ -0,0 +1,66 @@
+name: Build and Push OOTB Connectors Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Semantic version number of job worker runtime to use in release: ^[0-9]+.[0-9]+.[0-9]+(-[a-zA-Z0-9.-]+){0,1}$'
+        required: true
+      branch:
+        description: 'Git branch of Dockerfile to be used in the release'
+        default: 'main'
+      push-latest:
+        description: 'Push docker image with latest tag, i.e. this is the latest stable version released'
+        type: boolean
+        default: true
+
+jobs:
+  build-and-push:
+    name: Build and Push OOTB Connectors Docker Image
+    runs-on: ubuntu-latest
+    needs: [ prepare ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch }}
+
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@v2.4.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          exportEnv: false # we rely on step outputs, no need for environment variables
+          secrets: |
+            secret/data/products/connectors/ci/common DOCKERHUB_USER;
+            secret/data/products/connectors/ci/common DOCKERHUB_PASSWORD;
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ steps.secrets.outputs.DOCKERHUB_USER }}
+          password: ${{ steps.secrets.outputs.DOCKERHUB_PASSWORD }}
+
+      - name: Build and Push Docker Image tag ${{ github.event.inputs.version }}-ootb
+        uses: docker/build-push-action@v3
+        with:
+          context: runtime-job-worker/
+          file: runtime-job-worker/Dockerfile.ootb
+          build-args: RUNTIME_VERSION=${{ github.event.inputs.version }}
+          push: true
+          tags: camunda/connectors:${{ github.event.inputs.version }}
```

Do we need to tag it as OOTB? We'll override the base image otherwise, right?
```suggestion
          tags: camunda/connectors:${{ github.event.inputs.version }}-ootb
```